src/modules/Builder/features/PerformanceTasks/Unity/Unity.tsx

```diff
@@ -17,8 +17,11 @@ export const Unity = () => {
 
   const [file, setFile] = useState<File | null>(null);
 
-  const urlName = 'activities[0].items[0].config.file';
+  const activities = watch('activities');
+  const unityActivityIndex = activities.findIndex((a: { type: string }) => a.type === 'unity');
```

Comment:
This can fail if there's more than 1 unity activity

Reply:
It's not decided yet, but multiple activities could be added, each with a different config file.

You can use the `useCurrentActivity` hook to retrieve the correct activity.

```
const { fieldName } = useCurrentActivity();
...
const urlName = `${fieldName}.items.0.config.file`;
```